Dangerfile

```diff
@@ -1,5 +1,15 @@
 # frozen_string_literal: true
 
+# Skip Danger run if the opened pull request has "wip" label, [WIP] in the title, or status is Draft
+has_wip_label = github.pr_labels.any? { |label| label.include? 'wip' }
+has_wip_title = github.pr_title.include? '[WIP]'
+is_draft = github.pr_draft?
+
+if has_wip_label || has_wip_title || is_draft
+  message('Skipping Danger since PR is classed as Work in Progress')
+  return
```

Comment:
```suggestion
  return message('Skipping Danger since PR is classed as Work in Progress')
```

Does it work in a single line? 👀

Reply:
Thanks 🙏🏼 . Fixed in 7e57cc5674f5e5e9916ce4094835242822a179f1

Reply:
Hi, Doing this rubocop throws error `Lint/TopLevelReturnWithArgument: Top level return with argument detected.`. Reverting back to previous implementation.

Reply:
Oh, thanks for the sharing 🙏